README.md

```diff
@@ -19,11 +19,18 @@ $ cat /etc/gu/explainer.stage.conf
 stage=DEV
 ```
 
+Configure the composer route in nginx:
+
+```
+cd <path_of_dev_nginx>
+sudo ./setup-app.rb <path_of_explainer>/nginx/nginx-mapping.yml
```

Comment:
As a point, you should be able to run:
`<path_of_dev-nginx>/setup-app.rb ./nginx/nginx-mapping.yml`

Making it more project specific.